apps/olat-api/static/openapi.yaml

```
@@ -299,12 +296,12 @@ components:
       properties:
         id:
           type: string
-          example: LIVE_QUIZZES
+          example: LIVE_QUIZ
           description: Unique identifier for the activity type.
         title:
           type: string
-          example: Live Quiz Overview (5)
-          description: Title of activity (may include count in parentheses for overview types).
+          example: Live Quiz 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)